website/docs/docs/cloud/about-cloud/dark-mode.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "dbt Cloud UI dark mode"
+id: dark-mode
+description: "About dbt Cloud UI dark mode"
+sidebar_label: dbt Cloud UI dark mode
+---
```

Comment:
non blocker but i think it'd be super nice to add an image to the frontmatter (the trick i shared at our offsite) so users see dark mode visually when the url unfurls!